source/nodejs/adaptivecards-designer/src/designer-peers.ts

```diff
@@ -1196,6 +1225,9 @@ export abstract class DesignerPeer extends DraggableElement {
         }
 
         element.style.position = "absolute";
+        // Issue: I believe this stops us from interacting with the elements - this could be okay since it shouldn't be an issue in preview mode
+        // Ensure that the peer is in front of the card
+        element.style.zIndex = "10";
```

Comment:
Not sure if this is the responsible code, but I found a z-order issue:

![zorder](https://user-images.githubusercontent.com/16614499/180515912-deebcac9-f60c-42e4-b443-8be5ea310705.gif)

Reply:
Resolved the zIndex issue, but still need to look into why tryDrop isn't working correctly :)